lib/cfn-model/transforms/serverless.rb

```diff
@@ -145,31 +155,34 @@ def lambda_function(handler:,
 
       # Return the Event structure of a AWS::Lambda::Function as created
       # by Serverless transform
-      def transform_function_events(cfn_hash, serverless_function, function_name)
+      def transform_function_events(cfn_hash, serverless_function, function_name, with_line_numbers)
         serverless_function['Properties']['Events'].each do |_, event|
-          serverlessrestapi_resources(cfn_hash, event, function_name) if event['Type'] == 'Api'
+          serverlessrestapi_resources(cfn_hash, event, function_name, with_line_numbers) if \
+            (event['Type'].is_a?(String) && event['Type'] == 'Api') ||
```

Comment:
make this guy is in a predicate or using one of the other predicates from above

Reply:
Updated in latest commit.